src/Nethermind/Nethermind.Merge.Plugin/Handlers/V1/ForkchoiceUpdatedV1Handler.cs

```diff
@@ -107,21 +104,30 @@ public Task<ResultWrapper<ForkchoiceUpdatedV1Result>> Handle(ForkchoiceStateV1 f
                 {
                     _logger.Info($"Syncing... Unknown forkchoiceState head hash... Request: {requestStr}.");
                 }
-
                 return ForkchoiceUpdatedV1Result.Syncing;
             }
 
             BlockInfo blockInfo = _blockTree.GetInfo(newHeadBlock.Number, newHeadBlock.GetOrCalculateHash()).Info;
             if (!blockInfo.WasProcessed)
             {
+                BlockHeader? blockParent = _blockTree.FindHeader(newHeadBlock.ParentHash!);
+                if (blockParent == null)
+                {
+                    if (_logger.IsDebug)
+                        _logger.Debug($"Parent of block not available. Starting new beacon header. sync.");
+
+                    StartNewBeaconHeaderSync(forkchoiceState, newHeadBlock!, requestStr);
```

Comment:
Is it possible scenario?

Reply:
I'm guessing this could trigger it. https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Merge.Plugin/Handlers/V1/NewPayloadV1Handler.cs#L130